block_util/src/lib.rs

```diff
@@ -18,8 +18,10 @@ pub mod qcow_sync;
 pub mod raw_async;
 pub mod raw_sync;
 pub mod vhd;
+pub mod vhdx_sync;
```

Comment:
Just a comment about your commit message. Please replace `vhdx: vhdx crate integration with the cloud hypervisor` with `block_util, vmm: vhdx crate integration with the cloud hypervisor`.